.eslintrc.json

```diff
@@ -56,6 +58,18 @@
     ],
     "quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]
   },
+  "settings": {
+    "import/core-modules": 
+      [
+        "googlesitekit-data",
+        "googlesitekit-modules",
+        "googlesitekit-api",
+        "googlesitekit-widgets",
```

Comment:
I believe that we're removing these soon anyway

Reply:
Not the above `googlesitekit-` ones, no.

Reply:
Ok @aaemnnosttv 

I can see them in the webpack config e.g. `'googlesitekit-data': './assets/js/googlesitekit-data.js',`

It looks like the default import is always imported.  I could try and put all those aliases into `eslint-plugin-import` (and put the eslint resolver plugin back in)

Or we can just leave them like this and ignore them. I figure webpack will not build if this file changes

Reply:
Sorry, I don't quite understand this comment. These are being ignored because the ESLint plugin can't resolve the path, correct?

That seems fine, but are you saying you can add the paths to the `eslint-plugin-import` config? If so that'd be arguably better—you're right, if we remove these files the plugin/webpack build will blow up, but no harm in having ESLint be aware of these paths rather than ignoring them, if it's just as easy?

Reply:
Yes @tofumatt , have a look at this 

https://github.com/google/site-kit-wp/pull/3745/commits/701c180e324cb67e69ef92a1aa94835c74e2c2db#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR73

On a previous commit I installed `eslint-import-resolver-alias` and configured this for `test/js/utils` (but Evan said better to just make all those relative imports and drop this alias)

I should be able to just do some config like the following though (if you change your mind)

```
    "import/resolver": {
        "alias": [
          ["googlesitekit-data". "./assets/js/googlesitekit-data.js"]
 ```